inc/Config/QueryRunner/QueryRunner.php

```diff
@@ -40,16 +40,16 @@ protected function get_request_details( HttpQueryInterface $query, array $input_
 			return $headers;
 		}
 
-		$method = $query->get_request_method();
```

Comment:
This just moves things around to exit early, and only do url construction work after those initial checks pass.